src/app/components/Select/index.jsx

```diff
@@ -35,13 +37,15 @@ export default function SelectOption({
 				filterSort={(optionA, optionB) =>
 					optionA.children.toLowerCase().localeCompare(optionB.children.toLowerCase())
 				}
-				defaultValue={defaultValue}
+				defaultValue={defaultValue || null}
+				disabled={disabled}
+				value={value}
 			>
 				{options &&
 					options.map(option => (
-						<AntOption key={option.key} value={option.value} style={{ fontSize: '1rem' }}>
+						<Option key={option.key} value={JSON.stringify(option)} style={{ fontSize: '1rem' }}>
```

Comment:
Why do we have to store value like this?

Reply:
i set all data for option value i fetched before, i wanna get all them on select option, i don't wanna get an id and fetch data after select that it's waste times and i cant handle it easy way with data from owner api